docs/data/sdks/android/index.md

```diff
@@ -155,6 +155,24 @@ Accurate session tracking requires that you enable `enableForegroundTracking(ge
     val sameClient = Amplitude.getInstance("Andy_Client")
     ```
 
+#### Configuration
+
+???config "Configuration Options"
+    | <div class="big-column">Name</div>  | Description | Default Value |
+    | --- | --- | --- |
+    | `eventUploadPeriodMillis` | The amount of time SDK will attempt to upload the unsent events to the server or reach `eventUploadThreshold` threshold. | `30000` |
+    | `eventUploadThreshold` | SDK will attempt to upload once unsent event count exceeds the event upload threshold or reach `eventUploadPeriodMillis` interval.  | `30` |
```

Comment:
I remembered eventUploadPeriodMillis is different if you are not enabled the foreground tracking. Do we need to add that cases?

Reply:
@yuhao900914 I guess what you mentioned is session time out, which is covered in session configs.

Reply:
Ha, you are right.

Reply:
But should we include both situation?

Reply:
@yuhao900914 Do you mean to include both in the description? Yeah, that is intentional to prevent confusion like which one comes first.

Reply:
Oh, sorry about that. There has some confusion on my side.